samples/sensor/bh1749/nrf9160_pca20035.overlay

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
This file is no longer needed after the `bh1749@38` node has been moved to `nrf9160_pca20035_common.dts`.